packages/browser-integration-tests/suites/replay/dsc/test.ts

```diff
@@ -74,12 +81,13 @@ sentryTest(
       });
     });
 
-    const envHeader = await getFirstSentryEnvelopeRequest<EventEnvelopeHeaders>(page, url, envelopeHeaderRequestParser);
```

Comment:
This actually does `page.goto(url)` under the hood, so we were reloading the page, re-initializing replay.